### PR TITLE
Add support for bookings_comments

### PR DESCRIPTION
**Pull request by p55**
solves https://github.com/BookingSync/bookingsync-api/issues/90

### CODE REVIEW

File: lib/bookingsync/api/client/bookings_comments.rb [hunk 1]
```diff
@@ -0,0 +1,72 @@
+module BookingSync::API
+  class Client
+    module BookingsComments
```
Comment (Azdaroth) [inline lib/bookingsync/api/client/bookings_comments.rb]: The resource is actually `BookingComment`, not `BookingsComment` (it's not named like join model). Should be `BookingComments` here I think.

File: lib/bookingsync/api/client/bookings_comments.rb [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      # Returns bookings comments for the account and rentals user is authenticated with.
+      # @param options [Hash] A customizable set of options.
+      # @option options [Integer] rental_id: Rental ID that booking is assiociated with.
+      # @return [Array<BookingSync::API::Resource>] Array of comments.
+      #
+      # @example Get the list of comments for the current account
+      #   @api.bookings_comments
+      # => [{:links=>{:booking=>10}, :id=>1, :content=>"comment 1", :editable=>true, :created_at=>2016-04-18 12:06:15 UTC, :updated_at=>2016-04-18 12:06:15 UTC},
+      #     {:links=>{:booking=>11}, :id=>2, :content=>"comment 2", :editable=>true, :created_at=>2016-04-18 12:06:25 UTC, :updated_at=>2016-04-18 12:06:25 UTC}]
+      # @example Get the list of comments only for specific rental
```
Comment (Azdaroth) [inline lib/bookingsync/api/client/bookings_comments.rb]: Bookings comments for specific rental? For booking, no?

File: lib/bookingsync/api/client/bookings_comments.rb [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+      #   @api.bookings_comments(rental_id: 1)
+      # => [{:links=>{:booking=>10}, :id=>1, :content=>"comment 1", :editable=>true, :created_at=>2016-04-18 12:06:15 UTC, :updated_at=>2016-04-18 12:06:15 UTC}]
+      def bookings_comments(options = {}, &block)
+        paginate :booking_comments, options, &block
+      end
+
+      # Get a single bookings_comment
+      #
+      # @param bookings_comment [BookingSync::API::Resource|Integer] BookingsComment or ID of the bookings_comment.
```
Comment (Azdaroth) [inline lib/bookingsync/api/client/bookings_comments.rb]: `bookings_comment` -> `booking_comment`

Comment (Azdaroth) [inline lib/bookingsync/api/client/bookings_comments.rb]: Same thing in other places.

**Comment by Azdaroth** [comment]
Updating changelog as well would be nice.

File: lib/bookingsync/api/client/bookings_comments.rb (hunk 2 above)
Comment (p55) [inline lib/bookingsync/api/client/bookings_comments.rb]: Nope. Core scopes this to specific rental in fact.

Comment (Azdaroth) [inline lib/bookingsync/api/client/bookings_comments.rb]: yeah, there is `for_rentals` scope, but can't really imagine filtering booking_comments by rental_id :P

Comment (p55) [inline lib/bookingsync/api/client/bookings_comments.rb]: @Mandaryn any reason for rental scoping? https://github.com/BookingSync/BookingSync/commit/523caeed98ee5a213010379e323c672cfd1450e7

File: lib/bookingsync/api/client/booking_comments.rb [hunk 4]
```diff
@@ -0,0 +1,72 @@
+module BookingSync::API
+  class Client
+    module BookingComments
+      # List bookings comments
```
Comment (ZenCocoon) [inline lib/bookingsync/api/client/booking_comments.rb]: `booking comments`, same at the line bellow

**Pull request merged**